src/assets/_mixins.scss

```diff
@@ -0,0 +1,6 @@
+//colors
+$whiteText: rgb(238, 238, 238);
```

Comment:
Is it possible to use the hex here in preference over rgb()?

Reply:
Yes, I'll convert it to hex.